### PR TITLE
Add some missing fixtures

### PR DESCRIPTION
**Pull request by dixpac**
There were few missing fixtures, and we mix fixtures with direct
`ActiveRecord` calls, which is not bad for complex stuff..but these
could all be represented with the fixtures. It is better that we have
basic data loaded.

### CODE REVIEW

File: spec/features/fee_type_spec.rb [hunk 1]
```diff
@@ -2,34 +2,32 @@
...
-    @fee_type.save!
   end
 
   scenario 'SHOW' do
     visit fee_types_path
-    click_link @fee_type.name
+    click_link fee_type.name
 
     expect(page).to have_content('Upgrade Membership')
     expect(page).to have_content('20.00')
```
Comment (DamirSvrtan) [inline spec/features/fee_type_spec.rb]: From the spec here there is no way to see why the page would have '20.00' or 'Upgrade Membership' in it's content. Usually with factory bot i'd, i'd explicitly set those values in the spec before inserting them (I know that is not possible here).

Would it be better in this case actually to write `expect(page).to have_content(fee_type.name)`...

**Review by DamirSvrtan — APPROVED** [review]
Thanks @dixpac for doing this! Just had one comment but it's not a blocker!

File: spec/features/fee_type_spec.rb (hunk 1 above)
Comment (dixpac) [reply spec/features/fee_type_spec.rb]: 100% agree...I was concentrated just to fix/replace fixtures didn't have to to look inside `expectations`...will push the correction tomorrow morning.

**Comment by dixpac** [comment]
@DamirSvrtan  I've fixed and merged since you've approved yesterday...didn't want to wait and deal with possible conflicts :)

**Pull request merged**